homeassistant/components/braviatv/config_flow.py

```diff
@@ -262,7 +263,8 @@ async def async_step_init(
             self.config_entry.entry_id
         ]
 
-        await coordinator.async_update_sources()
+        with suppress(BraviaTVError):
```

Comment:
Isn't it better to abort the flow if there's an error?

Reply:
Yes, it was the most reasonable option, but the frontend doesn't display the dialog correctly.

<img width="530" alt="image" src="https://user-images.githubusercontent.com/11841379/200170499-bc949322-5cc7-4c93-9ad8-b464de0116a8.png">

Reply:
How is the dialog not displayed correctly?

Reply:
It seems to me that the heading is missing in the dialog. In addition, I did not find cases of using abortion from options flow and thought that it was not usually used there. If this is normal behavior then I will do it.

Reply:
It should be ok to use abort in any flow as far as I know. If the title is missing that's a separate issue, I think.

Reply:
If the integration is initialized then `coordinator.source_map` contains the source list, we just want to refresh. In addition, the error when the TV is turned off appears only for older TV models. IMO suppress is enough here.

Reply:
It looks like this. I think we need to report this in the frontend repo.

<img width="678" src="https://user-images.githubusercontent.com/11841379/200182216-fdc6bc39-1221-4561-a1bf-cddb1163610e.png">

Reply:
Yes, seems like a frontend issue.

I think it's better to abort if we can't update. Otherwise the user may get an unexpected result.